profile.d/security

```diff
@@ -26,3 +26,9 @@ RELEASE_GIT_PRODUCTION_BRANCH=master
 #         as retrieved from ./utils/getJenkinsVersion.py
 
 MAVEN_REPOSITORY_PRODUCTION_NAME=releases
+
+# Most settings above are unused
+RELEASE_GIT_BRANCH=security-stable-2.414
```

Comment:
should this be?
```suggestion
RELEASE_GIT_BRANCH=security-stable-2.414.2
```

Reply:
No, this a a branch name in `jenkinsci-cert/jenkins`